index/server/main.go

```diff
@@ -197,11 +198,35 @@ func main() {
 	router.HEAD("/v2/*proxyPath", ociServerProxy)
 	router.GET("/v2/*proxyPath", ociServerProxy)
 
+	router.Any("/viewer", serveUI)
+	router.Any("/viewer/*proxyPath", serveUI)
```

Comment:
technically we only need GET for /viewer and all its sub paths right?

Reply:
Yeah. I used `Any` in case the viewer needed to support other methods in the future, but we can always change it back if needed.

I'll switch to GET, since we really only need that.

Reply:
Fixed.